pytest_doctestplus/plugin.py

```diff
@@ -953,6 +960,9 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
     all_bad_tests = []
     if not diff_mode:
         return  # we do not report or apply diffs
+    
+    # get encoding to open file default ini=None or option="utf-8"
+    encoding = config.getini("text_file_encoding") or config.getoption("text_file_encoding")
```

Comment:
any specification what has the higher priority cli-option or ini-file?

Reply:
This seems to be more consistent with the rest of the plugin, but I have to realise we're not 100% consistent 😱

Reply:
should be solved with 4a639d7